com.markopejic.downloader.metainfo.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2024 Marko Pejić -->
+<component type="desktop-application">
+  <id>com.markopejic.downloader</id>
+  
+  <name>Media Downloader</name>
+  <summary>Web video/audio downloader</summary>
+
+  <content_rating type="oars-1.0" />
+  <url type="homepage">https://downloader.markopejic.com</url>
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+
+  <developer id="com.markopejic">
+    <name>Marko Pejić</name>
+  </developer>
+  <url type="vcs-browser">https://github.com/markop404/media-downloader</url>
+
+  <releases type="external">com.markopejic.downloader.releases.xml</releases>
```

Comment:
this is not supported. Put the releases inline.

Reply:
[Embedded](https://github.com/markop404/flathub/commit/4d0891db6692ef15d91c0ce440107f45dde0e30d) release information directly to com.markopejic.downloader.metainfo.xml.